launcher/cog-launcher.c

```diff
@@ -238,11 +238,14 @@ cog_launcher_create_view(CogLauncher *self, CogShell *shell)
     }
 
     g_autoptr(GError) error = NULL;
-    CogPlatform      *platform =
-        cog_platform_configure(s_options.platform_name, s_options.platform_params, "COG", shell, &error);
+    CogPlatform      *platform = cog_platform_new(s_options.platform_name ?: g_getenv("COG_PLATFORM_NAME"), &error);
```

Comment:
I think the this could be like this  🤔  :

```suggestion
    CogPlatform *platform = cog_platform_get_default();
    if (!platform) {
        platform = cog_platform_new(s_options.platform_name ?: g_getenv("COG_PLATFORM_NAME"), &error);
    }   
```